tests_external/bar.js

```diff
@@ -1 +1 @@
-exports.bar = 5
\ No newline at end of file
+export let bar = 5;
```

Comment:
Did you get any linting fail because of this? I think originally the idea was to check import attribute/expressions would work also with commonjs modules directly.

Reply:
No, the reason I changed it was to be able to generate and bundle all tests directly to/From ES6 modules. I'll change it back, but it's forcing all the tests to be compiled to commonjs just for this test.